bot/src/commands/index.js

```diff
@@ -0,0 +1,9 @@
+import { AddInboxItem } from './add-inbox-item';
+import { RevertLastCommand } from './revert-last-command';
+import { CreateReport } from './create-report';
+
+export const Commands = [
```

Comment:
Perhaps consider directly exporting the imports?

    export { AddInboxItem } from './add-inbox-item';
    // ..etc..

In `bot.js` you can then do:

    import * as commands from './commands';